test/functional/lua/vim_spec.lua

```diff
@@ -438,6 +438,13 @@ describe('lua stdlib', function()
     ]]))
   end)
 
+  it('vim.tbl_repeat', function()
+    eq({{1, 2, 3}, {1, 2, 3}, {1, 2, 3}, {1, 2, 3}},
+      exec_lua("return vim.tbl_repeat({1, 2, 3}, 4)"))
```

Comment:
I bet that if you do the following the test will yeild something interesting:
```
local b = vim.tbl_repeat({1, 2, 3}, 4)
b[1][1] = 4
```

Reply:
I intentionally didn't `deepcopy`, but I'm open to suggestions obviously -- you expect it to `deepcopy`, or is it enough to add a note saying it does not copy to the docs?

Reply:
I think the repeat this way is fine, just mention that in the docs to be sure.

Reply:
Done (and rebased), good catch, thanks.